src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
```

Comment:
Checklist point 2:
> 2. Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

So you can change literal number of days into some const value called for example long_term and short_term :)

Reply:
My new solution:

`const BASE_PRICE_PER_DAY = 40;
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;
const SHORT_TERM = 3;
const SHORT_TERM_DISCOUNT = 20;

function calculateRentalCost(numberOfDays) {
  const basePrice = BASE_PRICE_PER_DAY * numberOfDays;

  if (numberOfDays >= LONG_TERM) {
    return basePrice - LONG_TERM_DISCOUNT;
  }

  if (numberOfDays >= SHORT_TERM) {
    return basePrice - SHORT_TERM_DISCOUNT;
  }

  return basePrice;
}
`

Test Suites: 1 passed, 1 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        1.397s
Ran all test suites matching /.\//i.
Tests Passed

unfortunatlly:

To https://github.com/Capoverde/js_task-transportation-on-vacation.git
 ! [rejected]        develop -> develop (non-fast-forward)
error: failed to push some refs to 'https://github.com/Capoverde/js_task-transportation-on-vacation.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.